oozie-to-airflow/examples/fs/configuration.properties

```diff
@@ -0,0 +1,16 @@
+# Copyright 2019 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+dataproc_cluster=cluster-o2a
+gcp_region=europe-west3
```

Comment:
It's better not to commit `configuration.properties` files, only templates. Please add a `.gitignore` to this folder (feel free to take a look at other examples).

Reply:
I deleted this file.